.goreleaser.yml

```diff
@@ -15,5 +15,6 @@ archive:
   files:
     - LICENSE
     - plugin.yaml
+    - completion.yaml
```

Comment:
@marckhouzam No need to add this here. The completion file will be included in the  archive.

Reply:
I reverted it.